core/src/main/java/gyro/core/scope/RootScope.java

```diff
@@ -233,12 +239,12 @@ public Object convertValue(Type returnType, Object object) {
         return converter.convert(returnType, object);
     }
 
-    public List<Resource> findResources() {
-        return findResourcesIn(null);
+    public List<Resource> findSortedResources() {
+        return findSortedResourcesIn(null);
     }
 
-    public List<Resource> findResourcesIn(Set<String> diffFiles) {
-        Stream<Resource> stream = Stream.concat(Stream.of(this), getFileScopes().stream())
+    public List<Resource> findSortedResourcesIn(Set<String> diffFiles) {
```

Comment:
If this method is not going to be overloaded, why not keep the original name `findResourcesIn `? `findSortedResourcesIn` seems like would have an overidden method `findResourcesIn `.